### ISSUE TITLE
Get-DbaBackupHistory - Value was either too large or too small for an Int32

### ISSUE THREAD

**Issue by elfgirl** [sqlcollaborative/dbatools#3777]
### Before submitting a bug report:

- [ X] Ensure you are able to reproduce it on the latest released version (_we release often_)
- [X ] Verified this bug is not already reported in an issue
- [X ] Verified errors are not related to permissions
- [ X] Can reproduce in a clean PowerShell session (_clean = `powershell -NoProfile`_)

## Steps to Reproduce

Second DB in a list of Databases. Size is only ~417MB. First one is 67MB. First one retrieves backups fine, second one throws the below error. Oddly number shown does not match any size we can find on our size. 

```powershell
<#
   $prodBackup = Get-DbaBackupHistory -SqlInstance $prodSqlServer -Database $DatabaseName -LastFull
#>
```

## Expected Behavior

Size is returned and BackupHistory returns a list of files. It does for previous execution loop. 

## Actual Behavior

Exception             : System.Management.Automation.SetValueInvocationException: Exception setting "TotalSize": "Cannot convert value "2185653248" to type "Sqlcollaborative.Dbatools.Utility.Size". Error: "Value was either too large or toosmall for an Int32."" ---> System.Management.Automation.PSInvalidCastException: Cannot convert value "2185653248" to type "Sqlcollaborative.Dbatools.Utility.Size". Error: "Value was either too large or too small for an Int32." ---> System.OverflowException: Value was either too large or too small for an Int32.

## Environmental data

Name                           Value
----                           -----
PSVersion                      5.1.14409.1012
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14409.1012
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

 - - SQL Server: 
2012 (SP4-GDR)

**Comment by Zhunya** [comment]
@elfgirl what version you use - `(Get-Module dbatools).Version`

**Comment by Zhunya** [comment]
I think this fixed in #3504

**Comment by elfgirl** [comment]
I think you are right. After coming back to this, all DBs are restoring now.
